lib/rubocop/cops_documentation_generator.rb

```diff
@@ -183,9 +183,9 @@ def configurable_values(pars, name)
   # rubocop:enable Metrics/CyclomaticComplexity,Metrics/MethodLength
 
   def to_table(header, content)
-    table = ['|===', "| #{header.join(' | ')}\n\n"].join("\n")
+    table = ['[separator=¦]', '|===', "| #{header.join(' | ')}\n\n"].join("\n")
```

Comment:
I'd suggest adding above a short comment explaining why this is needed for future reference.

Reply:
Sure! I've updated it!